runners/direct-java/src/main/java/org/apache/beam/runners/direct/DirectTimerInternals.java

```diff
@@ -45,11 +47,24 @@ private DirectTimerInternals(
   }
 
   @Override
+  public void setTimer(StateNamespace namespace, String timerId, Instant target,
+      TimeDomain timeDomain) {
+    // TODO: actually store by ID
```

Comment:
On the other hand, it "works" in that there is currently no hook to receive a particular timer and it is only `deleteTimer` that is unsupported... Joking aside, I was going to port GABW to call this with a dummy ID so I could delete the other method.

But for this PR I've decided to take your advice. Maybe by the time I issue a PR against GABW, it will be supported :-)